lumen/ai/agents.py

```diff
@@ -225,6 +208,10 @@ async def respond(
 
 
 class ChatAgent(Agent):
+    """
+    ChatAgent provides general information about available data
+    and other topics  to the user.
+    """
```

Comment:
Wondering why only SourceAgent/ChatAgent adds a docstring? Is it used?

Reply:
Because I haven't finished writing docstrings.

Reply:
Sorry, to be clear, yes it's used, as a docstring, all classes should have docstrings.

Reply:
Can you clarify what Agent's docstrings are used for in FunctionTool?

Reply:
They aren't. They are just docstrings.

Reply:
Okay, I was confused about "to be clear, yes it's used"